packages/sdk-multichain/src/multichain/transports/default/index.ts

```diff
@@ -3,34 +3,63 @@ import type { CaipAccountId } from '@metamask/utils';
 import type { ExtendedTransport, RPCAPI, Scope, SessionData } from 'src/domain';
 import { addValidAccounts, getOptionalScopes, getValidAccounts } from 'src/multichain/utils';
 
+const DEFAULT_REQUEST_TIMEOUT = 60 * 1000;
```

Comment:
What is the basis for a 60 second timeout? Can you explain why we're adding this now? What is it fixing or protecting against specifically?